src/utils/insert_into_db.py

```diff
@@ -0,0 +1,53 @@
+import json
+from psycopg2.extras import execute_values
+
+def insert_into_db(conn, table_name, json_data, column_order):
+    try:
+        cursor = conn.cursor()
+
+        cursor.execute(f"""
+            SELECT column_name 
+            FROM information_schema.columns 
+            WHERE table_name = '{table_name}' 
+            AND column_default IS NULL
+            AND is_identity = 'NO'
+        """)
+        table_columns = [row[0] for row in cursor.fetchall()]
```

Comment:
Why are you fetching columns before insert?

Reply:
I did to check every column was introduced in the input "column_order" of the function. If the person who called function forgot one column, the program will throw an error and it'll finish.